<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DenyPrivilegesDialog</class>
 <widget class="QDialog" name="DenyPrivilegesDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>725</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2500</width>
    <height>3500</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Driving Privileges Information</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 61, 102);
font: 75 10pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(29, 61, 102);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(107, 107, 107);</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1169</width>
           <height>685</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
          <item row="3" column="0">
           <widget class="QPushButton" name="close_dialog_Button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 96, 82);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Close Dialog</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="create_entry_Button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 255, 0);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Open Entry</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QFrame" name="case_name_Frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <layout class="QGridLayout" name="gridLayout" columnstretch="0,2,0,1,1">
             <item row="4" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Def. First Name:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="4">
              <widget class="QPushButton" name="clear_fields_case_Button">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(160, 160, 160);</string>
               </property>
               <property name="text">
                <string>Clear Fields</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="defendant_first_name_lineEdit">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="defendant_last_name_lineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Def. Last Name:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="5">
              <widget class="QLabel" name="label_6">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Palatino Linotype</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                 <underline>true</underline>
                </font>
               </property>
               <property name="text">
                <string>CASE INFORMATION</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="2" colspan="2">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>Entry Date:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Case Number:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="4">
              <widget class="NoScrollDateEdit" name="plea_trial_date">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="minimumDate">
                <date>
                 <year>2021</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
               <property name="date">
                <date>
                 <year>2021</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
              </widget>
             </item>
             <item row="5" column="4">
              <widget class="QPushButton" name="cancel_Button">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(160, 160, 160);</string>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="case_number_lineEdit">
               <property name="maximumSize">
                <size>
                 <width>750</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="12" column="0">
              <widget class="QRadioButton" name="permit_renew_radio_btn">
               <property name="text">
                <string>Permit Defendant to Renew upon Expiration</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QRadioButton" name="permit_test_radio_btn">
               <property name="text">
                <string>Permit Defendant to Test and/or Renew</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="deny_privileges_radio_btn">
               <property name="text">
                <string>Deny Driving Privileges</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="petition_incomplete_check_box">
               <property name="text">
                <string>Petition is Incomplete and/or Illegible</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_4">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Palatino Linotype</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                 <underline>true</underline>
                </font>
               </property>
               <property name="text">
                <string>DRIVING PRIVILEGES ENTRY TYPE</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="hard_time_check_box">
               <property name="text">
                <string>Not Eligible - Hard Time has not passed</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="no_jurisdiction_check_box">
               <property name="text">
                <string>License not subject to suspension for which this court may grant privileges</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="prohibited_activities_check_box">
               <property name="text">
                <string>Petition requests privileges for activities not permitted by this Court</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="no_insurance_check_box">
               <property name="text">
                <string>Failed to Provide Proof of Insurance</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QCheckBox" name="no_employer_info_check_box">
               <property name="text">
                <string>Court could not verify employer information</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QDateEdit" name="license_exp_date_field">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="license_exp_date_label">
               <property name="text">
                <string>Defendant Driver License Expiration Date:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QCheckBox" name="permanent_id_check_box">
               <property name="text">
                <string>Check if defendant has permanent ID card instead of expired license</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0" colspan="2">
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="2">
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="out_of_state_license_check_box">
               <property name="text">
                <string>Out of State License - Needs Certified Abstract</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NoScrollDateEdit</class>
   <extends>QDateEdit</extends>
   <header>munientry.widgets.custom_widgets</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>defendant_first_name_lineEdit</tabstop>
  <tabstop>defendant_last_name_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>create_entry()</slot>
  <slot>continue_dialog()</slot>
  <slot>close_window()</slot>
  <slot>set_dialog()</slot>
  <slot>update_case_information()</slot>
  <slot>ovi_dialog()</slot>
  <slot>sentencing_dialog()</slot>
  <slot>proceed_to_sentencing()</slot>
  <slot>proceed_to_ovi()</slot>
  <slot>add_charge()</slot>
  <slot>set_statute()</slot>
  <slot>set_pay_date()</slot>
  <slot>amend_offense()</slot>
  <slot>set_offense()</slot>
  <slot>add_charge_to_view()</slot>
  <slot>delete_charge()</slot>
  <slot>delete_charge_from_view()</slot>
  <slot>close_event()</slot>
  <slot>set_fra_in_file()</slot>
  <slot>set_fra_in_court()</slot>
  <slot>start_add_conditions_dialog()</slot>
  <slot>start_amend_offense_dialog()</slot>
 </slots>
</ui>
